docs/theme-context.rst

```diff
@@ -0,0 +1,123 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the `html_context Sphinx setting`_ in
+the ``conf.py`` file. This extra context allows Read the Docs to add some
+features like "Edit on GitHub" and use a user custom Analytics code, among others.
+
+.. _html_context Sphinx setting: http://www.sphinx-doc.org/en/stable/config.html#confval-html_context
+
+Context injected by default
```

Comment:
"by default" implies there is a non-default injection, which isn't user configurable. I think this can be dropped from the heading